src/Generator/TraitUsageGenerator.php

```diff
@@ -11,6 +11,12 @@
 use Reflection;
 use ReflectionMethod;
 
+/**
+ * Class TraitUsageGenerator
+ *
+ * @deprecated Deprecated in 2.7. Use UsageGenerator instead
```

Comment:
I don't understand why this was changed, to be honest: `TraitUsageGenerator` is very specific and clear, whereas `UsageGenerator` seems too generic, and may be confused with the `use` statements generator at namespace level